build.gradle

```diff
@@ -10,7 +10,7 @@ apply plugin: "com.gradle.plugin-publish"
 
 archivesBaseName = 'liberty-gradle-plugin'
 group = 'io.openliberty.tools'
-version = '3.2.2-SNAPSHOT'
+version = '3.2.3-SNAPSHOT'
```

Comment:
This was supposed to be 3.3-SNAPSHOT. New functionality warrants a point release instead of a minor fix release.

Reply:
Also, you will need to rebase off the latest master to get the tests to pass.